tests/conftest.py

```diff
@@ -1,32 +1,110 @@
 # coding=utf-8
+from datetime import datetime, timedelta
+from unittest.mock import patch
+from uuid import uuid4
+
 import pytest
 from multidict import CIMultiDict
 
+from lbz.authentication import User
+from lbz.authz.authorizer import Authorizer
 from lbz.dev.misc import Event
 from lbz.request import Request
+from tests import SAMPLE_PRIVATE_KEY
+from tests.fixtures.rsa_pair import SAMPLE_PUBLIC_KEY
+from tests.utils import encode_token, allowed_audiences
 
+req = Request(
```

Comment:
Request in consider unmutable (except methods but they are not changing the data)

Reply:
Unfortunately, there are places where this rule is broken e.g. [test_resource.py](https://github.com/pdyba/lambdalizator/blob/master/tests/test_resource.py#L341). By using fixtures, we have a much clearer situation because it is impossible to change that 😉